### ISSUE TITLE
Failed to load the tool-bar package

### ISSUE THREAD

**Issue by dpadmor** [suda/tool-bar#314]
[Enter steps to reproduce:]

1. .Open Atom

**Atom**: 1.52.0 x64
**Electron**: 6.1.12
**OS**: Mac OS X 10.15.7
**Thrown From**: [tool-bar](https://github.com/suda/toolbar) package 1.4.0

### Stack Trace

Failed to load the tool-bar package

```
At '../dist/iconsets.css' wasn't found. Tried - /Users/dani/.atom/packages/tool-bar/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/node_modules/base16-tomorrow-dark-theme/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/node_modules/one-dark-ui/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/static/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/dist/iconsets.css,../dist/iconsets.css in /Users/dani/.atom/packages/tool-bar/styles/tool-bar.less:2:0

LessError: '../dist/iconsets.css' wasn't found. Tried - /Users/dani/.atom/packages/tool-bar/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/node_modules/base16-tomorrow-dark-theme/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/node_modules/one-dark-ui/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/static/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/dist/iconsets.css,../dist/iconsets.css
at /Users/dani/.atom/packages/tool-bar/styles/tool-bar.less:2:0
```

### Commands

```
```

### Non-Core Packages

```
compare-files 0.8.1 
ide-json 0.2.1 
language-docker 1.1.8 
language-markdown 0.37.0 
markdown-scroll-sync 2.1.2 
markdown-toc 0.4.2 
markdown-writer 2.11.10 
pp-markdown 0.0.2 
pretty-json 2.1.2 
tidy-markdown 3.0.1 
tool-bar 1.4.0 
tool-bar-markdown-writer 0.5.1 
xml-formatter 0.13.0 
```

**Comment by nhchan** [comment]
I created the same issue nearly at the same time. I've closed mine to relocate to add to this one:

[Steps to reproduce:]

1. Update to Atom 1.52.0, then
2. Restart Atom.
(Same issue after reinstalling tool-bar.)

**Atom**: 1.52.0 x64
**Electron**: 6.1.12
**OS**: Mac OS X 10.15.6
**Thrown From**: [tool-bar](https://github.com/suda/toolbar) package 1.4.0


### Stack Trace

Failed to load the tool-bar package

```
At '../dist/iconsets.css' wasn't found. Tried - /Users/echan/.atom/packages/tool-bar/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/node_modules/one-dark-syntax/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/node_modules/one-dark-ui/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/static/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/dist/iconsets.css,../dist/iconsets.css in /Users/echan/.atom/packages/tool-bar/styles/tool-bar.less:2:0

LessError: '../dist/iconsets.css' wasn't found. Tried - /Users/echan/.atom/packages/tool-bar/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/node_modules/one-dark-syntax/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/node_modules/one-dark-ui/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/static/dist/iconsets.css,/Applications/Atom.app/Contents/Resources/app.asar/dist/iconsets.css,../dist/iconsets.css
at /Users/echan/.atom/packages/tool-bar/styles/tool-bar.less:2:0
```

### Commands

```
```

### Non-Core Packages

```
hyperclick 0.1.5 
indent-detective 0.4.0 
ink 0.12.5 
julia-client 0.12.6 
language-julia 0.19.3 
latex-completions 0.3.6 
save-workspace 0.5.0 
Sublime-Style-Column-Selection 1.7.5 
tool-bar 1.4.0 
uber-juno 0.3.0 
```

**Comment by mspittler** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.48.0 x64
**Electron**: 5.0.13
**OS**: Linux Mint
**Thrown From**: [tool-bar](https://github.com/suda/toolbar) package 1.4.0


### Stack Trace

Failed to load the tool-bar package

```
At '../dist/iconsets.css' wasn't found. Tried - /home/marcus/.atom/packages/tool-bar/dist/iconsets.css,/usr/share/atom/resources/app.asar/node_modules/one-dark-syntax/dist/iconsets.css,/usr/share/atom/resources/app.asar/node_modules/one-dark-ui/dist/iconsets.css,/usr/share/atom/resources/app.asar/static/dist/iconsets.css,/usr/share/atom/resources/app.asar/dist/iconsets.css,../dist/iconsets.css in /home/marcus/.atom/packages/tool-bar/styles/tool-bar.less:2:0

LessError: '../dist/iconsets.css' wasn't found. Tried - /home/marcus/.atom/packages/tool-bar/dist/iconsets.css,/usr/share/atom/resources/app.asar/node_modules/one-dark-syntax/dist/iconsets.css,/usr/share/atom/resources/app.asar/node_modules/one-dark-ui/dist/iconsets.css,/usr/share/atom/resources/app.asar/static/dist/iconsets.css,/usr/share/atom/resources/app.asar/dist/iconsets.css,../dist/iconsets.css
at /home/marcus/.atom/packages/tool-bar/styles/tool-bar.less:2:0
```

### Commands

```
```

### Non-Core Packages

```
atom-ide-ui 0.13.0 
atom-latex 0.9.1 
autocomplete-bibtex 1.2.10 
busy-signal 2.0.1 
compare-files 0.8.1 
file-icons 2.1.43 
gitlab 0.1.3-7 
highlight-selected 0.17.0 
ide-python 1.5.0 
intentions 1.1.5 
language-latex 1.2.0 
language-pfm 0.82.0 
language-r 0.4.2 
latex 0.50.2 
linter 3.1.1 
linter-chktex 1.4.0 
linter-ui-default 2.1.2 
linter-write-good 0.9.3 
markdown-deluxe 1.0.0 
markdown-fold 0.3.0 
markdown-folding 0.3.1 
markdown-img-paste 0.4.0 
markdown-mindmap 0.6.2 
markdown-preview-plus 4.8.2 
markdown-table-formatter 2.9.2 
markdown-toc 0.4.2 
markdown-writer 2.11.10 
minimap 4.29.9 
multi-cursor 3.0.0 
open-file 2.1.0 
open-recent 5.0.0 
open-unsupported-files 1.0.20 
pandoc-convert 1.1.1 
pdf-view 0.72.0 
pen-paper-coffee-syntax 0.17.0 
platformio-ide-terminal 2.10.0 
process-palette 0.17.0 
project-manager 3.3.8 
sync-settings 5.0.5 
tool-bar 1.4.0 
tool-bar-markdown-writer 0.5.1 
wordcount 3.2.0 
Zen 0.18.0 
zotero-citations 5.2.0 
```

**Comment by suda** [comment]
Oh sorry for causing this issue! I just released `1.4.1` that should fix it!
